src/dfx/src/commands/start.rs

```diff
@@ -61,12 +61,12 @@ pub struct StartOpts {
     #[arg(long, conflicts_with = "pocketic")]
     enable_bitcoin: bool,
 
-    /// enable canister http requests
-    #[arg(long, conflicts_with = "pocketic")]
+    /// enable canister http requests (on by default for --pocketic)
```

Comment:
Is it not on by default for the replica, too?

Reply:
No. The reason it is a flag is because the outcalls adapter is a separate process which dfx doesn't start if it doesn't need to.